### ISSUE TITLE
[MNG-6778] Use https for schemaLocations

### ISSUE THREAD

**Issue by jira-importer** [apache/maven#8261]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** opened **[MNG-6778](https://issues.apache.org/jira/browse/MNG-6778?redirect=false)** and commented

All poms start with the following roottag:

```xml
<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```

The schemaLocation VALUE should be updated to `https://maven.apache.org/xsd/maven-4.0.0.xsd`; the key is just a constant String, even though it looks like a URL, better to keep this the same.

---

**Remote Links:**
- [GitHub Pull Request #299
](https://github.com/apache/maven/pull/299)

**Comment by jira-importer** [comment]
**[Christian Wansart](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cwansart)** commented

I'd like to take that task.

**Comment by jira-importer** [comment]
**[Christian Wansart](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=cwansart)** commented

Please have a look at https://github.com/apache/maven/pull/299
I did not change the other xsds. If you want, I can change them as well.

**Comment by jira-importer** [comment]
**[Robert Scholte](https://issues.apache.org/jira/secure/ViewProfile.jspa?name=rfscholte)** commented

Fixed in [d657c9c6b467db60c932799485fb4210c0bd1fe0](https://gitbox.apache.org/repos/asf?p=maven.git;a=commit;h=d657c9c6b467db60c932799485fb4210c0bd1fe0)
Thanks for the patch!
